comp/etw/impl/etwSession.go

```diff
@@ -91,8 +92,8 @@ func (e *etwSession) DisableProvider(providerGUID windows.GUID) error {
 	return ret
 }
 
-//export ddEtwCallbackC
-func ddEtwCallbackC(eventRecord C.PEVENT_RECORD) {
+//export etwCompCallbackC
+func etwCompCallbackC(eventRecord C.PEVENT_RECORD) {
```

Comment:
because this is being used in system probe.  It was conflicting with the "other" etw callback already in place for NPM.  Worse, it wasn't a compile time conflict.  Go let it compile, but then just sent the callbacks to the wrong place.

Reply:
sure